gravitee-rest-api-management/gravitee-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiSubscriptionsResource.java

```diff
@@ -79,23 +84,23 @@
 
     @SuppressWarnings("UnresolvedRestParam")
     @PathParam("api")
-    @ApiParam(name = "api", hidden = true)
+    @Parameter(name = "api", hidden = true)
     private String api;
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = "List subscriptions for the API",
-            notes = "User must have the READ_SUBSCRIPTION permission to use this service")
-    @ApiResponses({
-            @ApiResponse(code = 200, message = "Paged result of API's subscriptions", response = PagedResult.class),
-            @ApiResponse(code = 500, message = "Internal server error")})
+    @Operation(summary = "List subscriptions for the API", description = "User must have the READ_SUBSCRIPTION permission to use this service")
+    @ApiResponse(responseCode = "200", description = "Paged result of API's subscriptions",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = PagedResult.class)))
```

Comment:
Could you extend `PagedResult<SubscriptionEntity>` as you have done in other classes ?